### DISCUSSION TITLE
Custom Backgrounds

### DISCUSSION THREAD

**Discussion by agluck91** [blinksh/blink#1834] [Ideas]
Given any thought to custom backgrounds? 
If you have, adding a slider to change how promenent it is?

**Comment by agluck91** [comment]
I know this is aesthetic, but I would love to see this added. 
It is the little things that matter! :D @carloscabanero  Thoughts on this one? Worth doing? Would this be hard to implement?

**Comment by yury** [comment]
@agluck91 you mean image as background?

**Reply by agluck91** [reply]
Yes exactly, with a slider to darken or lighten it.
